### PR TITLE
Build pipeline

### PR DESCRIPTION
**Pull request by camrossi**
New build pipeline and minor bug fixes

### CODE REVIEW

**Comment by samiib** [comment]
I fixed the merge conflicts with Dev. 😸

File: terraform/modules/k8s_node/vms.tf [hunk 1]
```diff
@@ -104,7 +104,7 @@ resource "null_resource" "cluster" {
   count      = var.k8s_cluster.sandbox_status ? 0 : 1
   depends_on = [local_file.AnsibleInventory, local_file.AnsibleConfig, vsphere_virtual_machine.vm]
   provisioner "local-exec" {
-    command = "ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -b -i ${var.ansible_dir}/inventory/nodes.ini ${var.ansible_dir}/cluster.yml"
+    command = "ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -b -i ${var.ansible_dir}/inventory/nodes.ini ${var.ansible_dir}/cluster.yaml"
```
Comment (samiib) [inline terraform/modules/k8s_node/vms.tf:107]: This file is actually cluster.yml. Check in the ansible folder.

Comment (camrossi) [reply terraform/modules/k8s_node/vms.tf:107]: Let me re-name the file!

**Review by samiib — APPROVED** [review]
Looks great to me. 🥇

**Pull request merged**